### DISCUSSION TITLE
something that can help

### DISCUSSION THREAD

**Discussion by BlackHatzzz** [V1li/H4X-Tools#26] [Ideas]
Hi bro, it would also be nice if you put a 0 to each option so it looks nicer, it would just be static but it would look good
![2024-11-29_14-30](https://github.com/user-attachments/assets/8cded25d-50cc-4c5a-8d8c-f7885fb3fe84)

**Comment by vil** [comment]
Hey,

Thanks for the suggestion, it has been added now.
